react/features/welcome/components/native/settings/components/SettingsView.js

```diff
@@ -200,6 +199,20 @@ class SettingsView extends AbstractSettingsView<Props, State> {
             }
         };
 
+        let helpScreenRoute;
+        let termsScreenRoute;
+        let privacyScreenRoute;
+
+        if (this.props.isTabNavigatorScreen) {
```

Comment:
Why is this necessary? Should we make the screen a container itself and navigate to those routes within?

Reply:
SettingsNavigationContainer can be used as a container screen only inside ConferenceNavigationContainer. 
If we use it inside the WelcomePageTabs it kind of complicates thing with navigation and screen options.